modules/template/templates.csv

```diff
@@ -24,4 +24,5 @@ biospecimen_metadata_template,SysBio AD biospecimen metadata template schema,,"C
 file_annotation_template,AD file annotation template schema,,"Component, Filename, individualID, specimenID, isMultiSpecimen, assay, fileFormat, consortium, study, grant, resourceType, dataType, dataSubtype, metadataType, analysisType, isModelSystem, chromosome",,False,ManifestTemplate,,sysbio.metadataTemplates-ad.manifest,,,template
 individual_animal_metadata_template,SysBio individual animal metadata template schema,,"Component, individualID, individualIdSource, species, sexChromosome, dateBirth, dateDeath, sex, genotype, genotypeBackground, room, litter, matingID, treatmentDose, modelSystemName",,False,ManifestTemplate,,sysbio.metadataTemplates-individual.animal,,,template
 individual_animal_MODEL-AD_metadata_template,individual animal metadata template schema for MODEL-AD Consortium,,"Component, individualID, climbID, microchipID, birthID, matingID, individualIdSource, materialOrigin, sex, species, generation, dateBirth, ageDeath, ageDeathUnits, brainWeight, rodentWeight, rodentDiet, bedding, room, waterpH, treatmentDose, treatmentType, stockNumber, genotype, genotypeBackground, individualCommonGenotype, modelSystemName, officialName",,False,ManifestTemplate,,sysbio.metadataTemplates-modelad.individual.animal,,,template
-individual_human_metadata_template,SysBio AD individual human metadata template schema,,"Component, individualID, cohort, species, sex, race, ethnicity, yearsEducation, apoeGenotype, diagnosis, diagnosisCriteria, ageDeath, ageDeathUnits, causeDeath, mannerDeath, PMI, pmiUnits, Braak, CERAD, brainWeight, pH",,False,ManifestTemplate,,sysbio.metadataTemplates-ad.individual.human,,,template
\ No newline at end of file
+individual_human_metadata_template,SysBio AD individual human metadata template schema,,"Component, individualID, cohort, species, sex, race, ethnicity, yearsEducation, apoeGenotype, diagnosis, diagnosisCriteria, ageDeath, ageDeathUnits, causeDeath, mannerDeath, PMI, pmiUnits, Braak, CERAD, brainWeight, pH",,False,ManifestTemplate,,sysbio.metadataTemplates-ad.individual.human,,,template
+assay_neuropath_metadata_template,SysBio neuropath metadata template schema,,"Component, Filename, individualID, conservationMethod, brainWeight, INT, PMI, AMY_THAL, AMY_A, CERAD, AMY_ANY, Braak, B_Score, ADNC, REAG, mayoDX, LEWY_full, alt_LEWY_full, LEWY_grp, LEWY_ANY, CDLB, INFA, HEMOR, MICR, WMRCAL, WMR, WBVD, HS_L, HS_S, alt_HS_S, TDP43, TDP_3, TDP_5, CVD_S, CVD_ATH, CVD_ATH_ANY, CVD_ART, CVD_ART_ANY, CVD_CAA, CVD_CAA_ANY",,False,ManifestTemplate,,sysbio.metadataTemplates-assay.neuropath,,,template  "
```

Comment:
```suggestion
assay_neuropath_metadata_template,SysBio neuropath metadata template schema,,"Component, Filename, individualID, conservationMethod, brainWeight, INT, PMI, AMY_THAL, AMY_A, CERAD, AMY_ANY, Braak, B_Score, ADNC, REAG, mayoDX, LEWY_full, alt_LEWY_full, LEWY_grp, LEWY_ANY, CDLB, INFA, HEMOR, MICR, WMRCAL, WMR, WBVD, HS_L, HS_S, alt_HS_S, TDP43, TDP_3, TDP_5, CVD_S, CVD_ATH, CVD_ATH_ANY, CVD_ART, CVD_ART_ANY, CVD_CAA, CVD_CAA_ANY",,False,ManifestTemplate,,sysbio.metadataTemplates-assay.neuropath,,,template
```

Reply:
Try removing the extra quotation mark here and see if that fixes the schema conversion error?